packaging/rpm-ostree.spec.in

```diff
@@ -27,6 +27,13 @@ BuildRequires: rust
 # Embedded unit tests
 %bcond_with bin_unit_tests
 
+# This is copied from the libdnf spec
+%if 0%{?rhel} && ! 0%{?centos}
+%bcond_without rhsm
+%else
+%bcond_with rhsm
+%endif
```

Comment:
I think we also need https://github.com/rpm-software-management/libdnf/blob/6e8e825960e3f7ba6efeb2c780fc268ca9c1614f/libdnf.spec#L77-L79.

Reply:
Good catch, fixed!